x/migration/types/message_claim_morse_application_test.go

```diff
@@ -4,26 +4,90 @@ import (
 	"testing"
 
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
-	"github.com/pokt-network/poktroll/testutil/sample"
 	"github.com/stretchr/testify/require"
+
+	"github.com/pokt-network/poktroll/testutil/sample"
+	apptypes "github.com/pokt-network/poktroll/x/application/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
 )
 
+const testServiceId = "svc1"
+
 func TestMsgClaimMorseApplication_ValidateBasic(t *testing.T) {
 	tests := []struct {
 		name string
 		msg  MsgClaimMorseApplication
 		err  error
 	}{
 		{
-			name: "invalid address",
+			name: "invalid ShannonDestAddress",
 			msg: MsgClaimMorseApplication{
 				ShannonDestAddress: "invalid_address",
+				MorseSrcAddress:    sample.MorseAddressHex(),
+				MorseSignature:     "mock_signature",
```

Comment:
update `MsgClaimMorseApplication#ValidateBasic()` to include signature length check (like `MsgClaimMorseAccount).